src/formDesigner/components/Concepts.js

```diff
@@ -12,22 +12,24 @@ class Concepts extends Component {
         { title: 'DataType', field: 'dataType' },
         { title: 'UUID', field: 'uuid', type: 'numeric' },
       ],
-      finalData : []
+      data : []
     }
   }
 
   componentDidMount(){
     axios
-    .get("/concept?size=10")
+    .get("/concept?size=1000") //Need to change API here to get user specific concepts
     .then((response)=>{
         const result = response.data._embedded.concept;
         const data1 = [];
```

Comment:
data1 will be better named as nonVoidedConcepts since that is what this variable contain.